homeassistant/components/acmeda/helpers.py

```diff
@@ -1,13 +1,21 @@
 """Helper functions for Acmeda Pulse."""
-from homeassistant.core import callback
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers.device_registry import async_get_registry as get_dev_reg
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from .const import DOMAIN, LOGGER
 
 
 @callback
 def async_add_acmeda_entities(
-    hass, entity_class, config_entry, current, async_add_entities
+    hass: HomeAssistant,
+    entity_class: type,
+    config_entry: ConfigEntry,
+    current: set[str],
```

Comment:
This is where the set is used.
unique_id is defined in line 27: `for unique_id, roller in api.items()`
nd then added to current on line 31: `current.add(unique_id)`

Reply:
I have traced it to an integer in `aiopulse`: `roller_id, ptr = utils.unpack_int(message, ptr, 6)`
See https://github.com/atmurray/aiopulse/blob/0d6dc539b739c5ea22a8729293526edb3090536c/aiopulse/hub.py#L270-L278

Reply:
Yes, looks right to me.